qa/suites/fs/basic_functional/tasks/acls-fuse-client.yaml

```diff
@@ -0,0 +1,4 @@
+tasks:
+  - cephfs_test_runner:
```

Comment:
almost forgot, and:

`  fail_on_skip: false`

See also `qa/suites/fs/basic_functional/tasks/client-recovery.yaml`.